migration-engine/migration-engine-tests/src/sql/quaint_result_set_ext.rs

```diff
@@ -91,8 +91,9 @@ impl<'a> RowAssertion<'a> {
             .and_then(|col: &Value<'_>| col.as_f64())
             .expect("Failed to extract f64");
 
+        let error_margin = f64::EPSILON;
         anyhow::ensure!(
-            actual_value == expected_value,
+            (actual_value - expected_value) < error_margin,
```

Comment:
We are expecting the value to roundtrip here, do we really want this?

Reply:
You're right; we could just leave it as-is and disable clippy for this instance.

Reply:
I'd be more comfortable with that, since we are not doing arithmetic on these floats, they should roundtrip. The rest lgtm's!